harness/determined/experimental/core_v2/_core_v2.py

```diff
@@ -22,7 +23,7 @@
 @dataclasses.dataclass
 class DefaultConfig:
```

Comment:
might make sense to rename this to just "Config" as well, since we've renamed DefaultConfig/UnmanagedConfig

Reply:
I'm not sure about renaming this class, isn't that the same as removing this public class?

Reply:
you can rename this one but create another class called `DefaultConfig` that's basically a dummy class with an `__init__` that logs a deprecation warning.

Reply:
Got it, thanks!